python/README.rst

```diff
@@ -1,18 +1,12 @@
 Vowpal Wabbit Python Wrapper
 ============================
 
-.. image:: https://badge.fury.io/py/vowpalwabbit.svg
-    :alt: PyPI Package
-    :target: https://pypi.python.org/pypi/vowpalwabbit
-.. image:: https://travis-ci.org/VowpalWabbit/vowpal_wabbit.svg?branch=master
-    :alt: Build Status
-    :target: https://travis-ci.org/VowpalWabbit/vowpal_wabbit
-.. image:: https://ci.appveyor.com/api/projects/status/6hqpd9e64h72gybr/branch/master?svg=true
-    :alt: Windows Build Status
-    :target: https://ci.appveyor.com/project/VowpalWabbit/vowpal-wabbit
-.. image:: https://coveralls.io/repos/github/VowpalWabbit/vowpal_wabbit/badge.svg
-    :alt: Coverage
-    :target: https://coveralls.io/r/VowpalWabbit/vowpal_wabbit
+|pic1| |pic2| |pic3| |pic4|
+
+.. |pic1| image:: https://badge.fury.io/py/vowpalwabbit.svg
```

Comment:
These are no longer clickable badges now though?

Reply:
They should still be clickable

Reply:
Wont clicking it take you to https://badge.fury.io/py/vowpalwabbit.svg ?

Reply:
Readded the link